authority/provisioners.go

```diff
@@ -494,9 +494,34 @@ func optionsToCertificates(p *linkedca.Provisioner) *provisioner.Options {
 			}
 		}
 	}
+	for _, wh := range p.Webhooks {
+		whCert := webhookToCertificates(wh)
+		ops.Webhooks = append(ops.Webhooks, whCert)
+	}
 	return ops
 }
 
+func webhookToCertificates(wh *linkedca.Webhook) *provisioner.Webhook {
```

Comment:
We probably need a webhookToLinkedca too.

Reply:
Where would that be used?